### PR TITLE
Story/vp 367/activities vertical menu

### PR DESCRIPTION
**Pull request by waltzaround**

### CODE REVIEW

File: components/Act/ActDetail.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ export function ActDetail ({ act }) {
           <Spacer />
         </Left>
         <Right>
-          <img style={{ width: '100%' }} src={img} alt={act.title} />
+          <img style={{ width: '100%' }} src={act.imgUrl} alt={act.title} />
```
Comment (MinhPhu0304) [inline components/Act/ActDetail.js]: I think this should be img variable instead. Because if the activity does not have image, we will use default image

Comment (MoniqueWarrington) [reply components/Act/ActDetail.js]: I think Walts code is primarily styling? It might be @alisamhh who wrote the logic <3

Comment (MoniqueWarrington) [reply components/Act/ActDetail.js]: I think there is a default fallback for if an activity/opportunity doesn't have an image

Comment (waltzaround) [reply components/Act/ActDetail.js]: Yerp, @alisamhh wrote the logic

**Pull request merged**